BEIMA.Backend/DeviceFunctions/AddDevice.cs

```diff
@@ -34,6 +35,14 @@ public static class AddDevice
             log.LogInformation("C# HTTP trigger function processed a device post request.");
 
 
+            var authService = AuthenticationDefinition.AuthticationInstance;
+            var claims = authService.ParseToken(req);
+
+            if(claims == null)
+            {
+                return new ObjectResult(Resources.UnauthorizedMessage) { StatusCode = 401 };
```

Comment:
If we aren't going to include a response message, we might as well just use an UnauthorizedResult object instead:
```return new UnauthorizedResult();```

Reply:
The unauthorized message gets returned as a string though. I thought that was the point of doing it this way instead of returning `UnauthorizedResult();`

Reply:
Right nvm, I didn't notice the Resources.UnauthorizedMessage you passed in, ignore me lol